### ISSUE TITLE
Alphabetisch browsen nach Titel - Artikel am Anfang des Titels

### ISSUE THREAD

**Issue by IxTheoKm** [ubtue/tuefind#833]
**Sollen die Artikel generell (nicht) berücksichtigt werden?**

Suchen Nutzer, die konkrete Titel kennen, nicht eher mit dem Artikel?

Offensichtlich sind Sortierzeichen derzeit relevant, da ich beim alphabetischen Browsen von Titeln den Artikel mit eingeben muss, wenn das Sortierzeichen fehlt. Titel mit Sortierzeichen finde ich nur unter dem folgenden Wort. 

Das hat zur Folge, dass gleiche Titel, wenn sie einmal manuell mit Sortierzeichen erfasst, und zusätzlich als elektronische Ressource (z.B. von DeGruyter) abweichend ohne Sortierzeichen eingespielt wurden, an verschiedenen Stellen stehen.

Erfassung mit Sortierzeichen vor "impur":
PPN 1644694409 und 1645305686

![browse4](https://user-images.githubusercontent.com/17004223/62762847-64265e00-ba8a-11e9-8b50-f0093e7d7b99.JPG)

Erfassung ohne Sortierzeichen vor "impur":
PPN 1523196262 und 1523196033

![browse5](https://user-images.githubusercontent.com/17004223/62762861-68eb1200-ba8a-11e9-891c-1007b3b0959d.JPG)

**Comment by thefass** [comment]
M.E sollten die Artikel berücksichtigt werden. 
Wir sollten das Issue auf der nächsten Sitzung diskutieren und dann entscheiden

**Comment by obrandt** [comment]
Artikel werden mit berücksichtigt.

**Comment by jriedl** [comment]
Die Extraktion der Titel erfolgt aus 245, womit Artikel mit berücksichtigt werden. Die vier o.g. PPNs folgen bei einer Suche nach "L'impur" unmittelbar aufeinander. Wie es scheint, liegt damit das gewünschte Verhalten bereits im aktuellen Stand vor. Ist die Problematik damit behoben?

**Comment by IxTheoKm** [comment]
Ja, das Verhalten ist jetzt anders als am 9. August beschrieben und damit wie gewünscht!
Ich schließe daher.

**Comment by IxTheoKm** [comment]
Offensichtlich sortieren die Artikel nun wieder _nicht_ mit, wie es im September 2019 schon mal umgesetzt war (s.o.)
Oder wurde nur "L' " entspr. umgesetzt? Die Regel müsste dann aber doch alle Artikel betreffen. Bei L' scheint es zu funktionieren, bei anderen Artikeln nicht:

https://ptah.ub.uni-tuebingen.de/Alphabrowse/Home?source=title&from=impur&botprotect=

![sortImp3](https://user-images.githubusercontent.com/17004223/86892546-afb11400-c100-11ea-9de9-58eb03c0ee98.JPG)

https://ptah.ub.uni-tuebingen.de/Alphabrowse/Home?source=title&from=limpur&botprotect=

![sortImp4](https://user-images.githubusercontent.com/17004223/86893102-6e6d3400-c101-11ea-9a12-1690ac81a00d.JPG)

**Comment by mtrojan-ub** [comment]
Aufgrund der heutigen Änderung in #874 müsste die Situation hinsichtlich `L' ...` ab morgen auf ptah wieder so sein wie am 11.9.2019.
Hat sich das Thema damit erledigt?

Mir ist allerdings noch folgendes aufgefallen (Aussagen aus #874):
- 08.07.2020: Grundsätzlich sollten alle Artikel gleich behandelt werden (ob beim Browsen oder in der Sortierung der Kurzliste)

Aktuell ist im Alphabetischen Browsen ein anderes Sortierfeld im Einsatz als bei den Titeln. Müsste das dann nicht eigentlich wieder vereinheitlicht werden, so dass das Browsen das gleiche Sortierfeld verwendet?
